test/Concurrency/transfernonsendable_nonisolatedunsafe.swift

```diff
@@ -507,6 +507,7 @@ enum NonIsolatedUnsafeComputedEnum: Sendable {
   case second
 
   nonisolated(unsafe) var nonIsolatedUnsafeVarObject: NonSendableKlass { NonSendableKlass() }
+  // expected-warning@-1{{}} 'nonisolated(unsafe)' has no effect on property 'nonIsolatedUnsafeVarObject', consider using 'nonisolated'
```

Comment:
I assume you meant to put the warning text inside the curly braces. Same for the change just below this one on line 609

Reply:
oh boy... yep